met-api/src/met_api/services/language_service.py

```diff
@@ -24,22 +23,6 @@ def get_languages():
         languages_records = Language.get_languages()
         return LanguageSchema(many=True).dump(languages_records)
 
-    @staticmethod
-    def create_language(language_data):
-        """Create language."""
```

Comment:
If we can delete languages, we should probably still be able to add them as well, otherwise the number of languages can only decrease with time. Is there a reason all this code was removed?

Reply:
If I left in a call to deleting languages themselves then I'll be removing it. Great point